src/components/shared/Button/PaneToggle.css

```diff
@@ -19,7 +19,7 @@ html[dir="rtl"] .toggle-button-start svg {
 }
 
 .toggle-button-end.vertical svg {
-  transform: rotate(-90deg);
+  transform: rotate(-90deg) !important;
```

Comment:
We could avoid "!important" by modifying selector to `html .toggle-button-end.vertical svg`